packages/esm-patient-common-lib/src/pagination/pagination.component.tsx

```diff
@@ -31,7 +34,13 @@ export const PatientChartPagination: React.FC<PatientChartPaginationProps> = ({
   return (
     <>
       {totalItems > 0 && (
-        <div className={isTablet ? styles.tablet : styles.desktop}>
+        <div
+          className={classNames(
+            isTablet ? styles.tablet : styles.desktop,
+            styles.paginationContainer,
+            grey ? styles.greyBackground : '',
+          )}
```

Comment:
One reason to use `classnames` is because it supports this ternary thing kind of natively, e.g.

```suggestion
          className={classNames(styles.paginationContainer, {
          	[styles.tablet]: isTablet,
            [styles.desktop]: !isTablet ,
            [styles.greyBackground]: grey
          })}
```